charts/newrelic-infrastructure-v3/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: newrelic-infrastructure-v3
 description: A Helm chart to deploy the New Relic Kubernetes monitoring solution
-version: 3.0.21
```

Comment:
Should we bump the minor if the app minor was updated?

Reply:
so far till is beta we were bumping patch only, but you are right, better the minor